src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorCSharpSemanticTokensInterceptor.cs

```diff
@@ -39,7 +39,7 @@ public RazorCSharpSemanticTokensInterceptor(LSPRequestInvoker requestInvoker)
         {
             var refreshParams = new SemanticTokensRefreshParams();
             await _requestInvoker.ReinvokeRequestOnServerAsync<SemanticTokensRefreshParams, Unit>(
-                LanguageServerConstants.RazorSemanticTokensRefreshEndpoint,
+AspNetCore.Razor.LanguageServer.Common.RazorLanguageServerCustomMessageTargets.RazorSemanticTokensRefreshEndpoint,
```

Comment:
Indentation, and possibly a using statement?

Reply:
interesting. This means the move code action left this unsimplified